### ISSUE TITLE
Support restore backup

### ISSUE THREAD

**Issue by ghost** [ZettaIO/restic-compose-backup#35]
It could be nice to support restore backups. Something like: rcb restore snapshotID

**Comment by einarf** [comment]
Yep. That would be nice. The plan was to do some code cleanup before adding new features. Things needs to be a bit more "pluggable" so it's easy to extend different types of backup. That will make restore a bit cleaner to deal with.

The project is still kind of in "prototype mode" meaning the design is very suboptimal.
